python/Converters/PycBasicData.h

```diff
@@ -99,19 +99,24 @@ namespace casacore { namespace python {
       PyObject* obj_ptr,
       boost::python::converter::rvalue_from_python_stage1_data* data)
     {
-#ifdef IS_PY3K
-    PyObject * temp_bytes = PyUnicode_AsEncodedString(obj_ptr, "ASCII", "strict"); // Owned reference
-    char* value;
-    if (temp_bytes != NULL) {
-        value = PyBytes_AS_STRING(temp_bytes); // Borrowed pointer
-        value = strdup(value);
-        Py_DECREF(temp_bytes);
+    char* value = 0;
```

Comment:
Personally I dislike `0` as a null pointer constant, and prefer `NULL` (or `nullptr` in C++11).

Reply:
changed